rules/windows/process_creation/proc_creation_win_reg_lsa_ppl_protection_disabled.yml

```
@@ -1,21 +1,23 @@
-title: Registry Disabling LSASS PPL
+title: LSA PPL Protection Disabled Via Reg
```

```suggestion
title: LSA PPL Protection Disabled Via Reg.EXE
```
You did it like that in another rule. I think in this case .EXE makes it more readable/understandable